src/server.c

```diff
@@ -5046,8 +5046,8 @@ sds genRedisInfoString(const char *section) {
             "active_defrag_misses:%lld\r\n"
             "active_defrag_key_hits:%lld\r\n"
             "active_defrag_key_misses:%lld\r\n"
-            "total_active_defrag_exceeded_time:%lld\r\n"
-            "current_active_defrag_exceeded_time:%lld\r\n"
+            "total_defrag_exceeded_time:%lld\r\n"
+            "current_defrag_exceeded_time:%lld\r\n"
```

Comment:
sorry for the zig-zag, still fishing, how about:
```suggestion
            "total_defrag_running_time:%lld\r\n"
            "current_defrag_running_time:%lld\r\n"
```
i.e. we have the `active_defrag_running` field in INFO MEMORY, and these stats are the history about how much it was actually running.

in eviction, we don't have a field that signifies that eviction is currently "running" i.e. that we're currently above the limit, for defrag we do.
so for eviction (and also OOM in case there's no eviction) the terminology is "exceeded", but for defrg, it's "running".

WDYT?

Reply:
I think `exceeded` is not good. But `running` still has problem. For `running`, I will think about `activeDefragCycle` execution time.

Reply:
@yossigo @madolson any suggestion for terminology?

Reply:
Exceeded time is definitely not great here, since I have no idea what it means without reading the code. I also have the same reaction that "active_defrag_running" is the the total amount of time spent actually doing the defrag work. Maybe a term like session could help bridge the gap?
```
current_defrag_session_time?
total_defrag_session_time?
```
Otherwise I would probably say we should just reinforce the terminology we are already using, running_time, and go with Oran's proposal.

Reply:
To me, `current_active_defrag_time` and `total_active_defrag_time` seem clear. I'm a bit reluctant to introduce "session" here because it's a bit out of context and confusing. The only thing I could still be confused about as a user is whether "time" is wall clock or CPU - but even the best name is no substitute to RTFM.

Reply:
how about `current_active_defrag_time_spent` and `total_active_defrag_time_spent`?
or `current_active_defrag_spent_time` and `total_active_defrag_spent_time`?

Reply:
actually then when i think of renaming `total_eviction_exceeded_time` to `total_eviction_spent_time` i think that's a bad idea. so maybe a vague term like `total_active_defrag_time` is indeed better.

Reply:
I don't think we have a good option here, so I'm pretty much fine with any of the options and we can document it clearly.